src/queries/__tests__/text.test.tsx

```diff
@@ -342,6 +342,28 @@ describe('supports TextMatch options', () => {
     expect(getAllByTestId('test', { exact: false })).toHaveLength(2);
   });
 
+  test('getByHintText, getByHintText', () => {
+    const { getByHintText, getAllByHintText } = render(
+      <View>
+        <View accessibilityHint="test" />
```

Comment:
I think these tests should be put into `hintText.test.tsx` file instead of here. Also add similar tests for `*ByLabelText` in `labelText.test.tsx`.